common/defs/fido/coinbase.json

```diff
@@ -0,0 +1,10 @@
+{
+  "name": "Coinbase",
+  "u2f": [
+    {
+      "app_id": "e27d61b4e99de0ed98163cb38b7af933c6665e5509e8490837055813778e236a",
+      "label": "coinbase.com"
+    }
+  ],
```

Comment:
I think you can drop this part (as can be seen from `core/src/apps/webauthn/knownapps.py` where exactly the same block is emitted twice)

Reply:
good point but in that case this modification is a no-op and we'll see the same result with TT

Reply:
Yep, for testing purpose it is fine to keep it.

Reply:
I think the "u2f" can be dropped. IIRC it is implied by the "webauthn".